tests/e2e/scripts/keadm_e2e.sh

```diff
@@ -45,10 +51,14 @@ function prepare_cluster() {
 
 function start_kubeedge() {
   sudo mkdir -p /var/lib/kubeedge
+  sudo mkdir -p /etc/kubeedge
   cd $KUBEEDGE_ROOT
   export KUBECONFIG=$HOME/.kube/config
 
-  sudo -E _output/local/bin/keadm init --kube-config=$KUBECONFIG --advertise-address=127.0.0.1
+  sudo cp _output/release/${VERSION}/kubeedge-${VERSION}-linux-amd64.tar.gz _output/release/${VERSION}/checksum_kubeedge-${VERSION}-linux-amd64.tar.gz.txt /etc/kubeedge
+
+  kubeedge_version=${VERSION: 1}
```

Comment:
What's meaning of this : 1?

Reply:
```
kubeedge::version::get_version_info
VERSION=${GIT_VERSION}
```
using this way, we got the VERSION is like `v1.9.1`, with a prefix `v`
But in `keadm init/join` cmd, we need to emphasize the kubeedge version using flag `--kubeedge-version=1.9.1`, but should not start with `v` prefix.